webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/uploading/UploadService.scala

```diff
@@ -481,8 +482,9 @@ class UploadService @Inject()(dataSourceService: DataSourceService,
         _ <- Fox.successful(())
         uploadedDataSourceType = guessTypeOfUploadedDataSource(unpackToDir)
         _ <- uploadedDataSourceType match {
-          case UploadedDataSourceType.ZARR | UploadedDataSourceType.NEUROGLANCER_PRECOMPUTED |
-              UploadedDataSourceType.N5_MULTISCALES | UploadedDataSourceType.N5_ARRAY =>
+          case UploadedDataSourceType.ZARR | UploadedDataSourceType.ZARR3 |
+              UploadedDataSourceType.NEUROGLANCER_PRECOMPUTED | UploadedDataSourceType.N5_MULTISCALES |
+              UploadedDataSourceType.N5_ARRAY =>
```

Comment:
Isn't ZARR§_MULTILAYER missing below in line 491?

Reply:
Same for within `tryExploringMultipleLayers`

Reply:
Absolutely right! I added it now.